GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -59,6 +63,19 @@ public class Glia {
 
     var stringProvidingPhase: StringProvidingPhase = .notConfigured
 
+    var maximumUploads: Int {
+        var value = Self.maximumUploads
+        /// `MAXIMUM_UPLOADS_PER_MESSAGE` is used in acceptance tests to decrease the time
+        /// for uploading maximum number of files in a single message.
+        let env = ProcessInfo.processInfo.environment
+        if let stringValue = env["MAXIMUM_UPLOADS_PER_MESSAGE"] {
+            /// Round to 1, because the attachment button doesn't turns to disabled state
```

Comment:
I don't understand the issue deeply, but isn't this a workaround rather than fixing the button's behavior?

Reply:
There was no issue, I just want to make `maximumUploads` value adjustable from acceptance tests. Because now uploading 25 files just to check whether attachmentButton turns to disabled state is a bit overhead. With 25 files the test take 10min, but with 5 files - 3mins. 

The problem with rounding to `0` exists, if we pass 0 or negative value to `MAXIMUM_UPLOADS_PER_MESSAGE` from acceptance tests. If we round to `0`, attachmentButton will be still enabled, due to missing any update events on that time. Nevertheless, we don't need to have this value set to `0`. That's why I didn't fix "rounding to 0"

Reply:
process information can't be changed after process is run, so I'd propose to change a little bit this getter and make it as `lazy var` and set value once instead of doing all this stuff for each get. would it make sense for you?

Reply:
Made it `lazy` b1447e9e5f7bafe28a9551b5ce7f4dc6f06a67e3